### PR TITLE
MM-34782 Cypress/E2E: Fix test for forgot password based on new email template

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix test for forgot password based on new email template

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-34782

#### Screenshots
Screen Shot 2021-04-13 at 11 28 07 AM

### CODE REVIEW

**Review by josephbaylon — APPROVED** [review]
LGTM. Tests passed locally.

File: e2e/cypress/integration/signin_authentication/forgot_password_spec.js [hunk 1]
```diff
@@ -56,25 +62,24 @@ function verifyForgotPasswordEmail(response, toUser, config) {
...
-    expect(bodyText[11]).to.equal(`The ${config.TeamSettings.SiteName} Team`);
-    expect(bodyText[13]).to.equal('To change your notification preferences, log in to your team site and go to Account Settings > Notifications.');
+    expect(bodyText.length).to.equal(11);
+    expect(bodyText[0]).to.equal('Reset Your Password');
+    expect(bodyText[1]).to.equal('Click the button below to reset your password. If you didn’t request this, you can safely ignore this email.');
+    expect(bodyText[3]).to.contain(`Reset Password ( ${SiteURL}/reset_password_complete?token=`);
+    expect(bodyText[5]).to.contain('The password reset link expires in 24 hours.');
+    expect(bodyText[7]).to.contain('Questions?');
+    expect(bodyText[8]).to.equal(`Email us any time at ${SupportEmail} ( ${SupportEmail} )`);
+    expect(bodyText[10]).to.equal('© 2021 Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA, 94301');
```
Comment (furqanmlk) [inline e2e/cypress/integration/signin_authentication/forgot_password_spec.js:82]: we will need to update this check every year :)

**Review by furqanmlk — APPROVED** [review]
All look good 
Passed locally 
image

One question though

**Review by prapti — APPROVED** [review]
LGTM. Test passed.

**Pull request merged**